docs/source/api/data.rst

```diff
@@ -0,0 +1,8 @@
+*****
```

Comment:
You need to add new files to the toctree in `source/api.rst`, or they won't show up in the documentation.